plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4402,6 +4402,16 @@ public void testSelectWithMoreThanOneSnapshotOfTheSameTable()
         assertUpdate(format("DROP TABLE %s", tableName));
     }
 
+    @Test
+    public void testInsertingIntoTablesWithColumnsWithQuotesInName()
+    {
+        String tableName = "tesT_inserting_into_tables_with_quotes_" + randomTableSuffix();
+        assertUpdate(format("CREATE TABLE %s (\"an identifier with \"\"quotes\"\" \" INTEGER, a row (\"another identifier\" INTEGER))", tableName));
```

Comment:
Let's call the row `x` since "a row" sounds like English and blends in with the other identifiers.